.drone.star

```diff
@@ -36,6 +36,17 @@ PYTHON = "python"
 THEGEEKLAB_DRONE_GITHUB_COMMENT = "thegeeklab/drone-github-comment:1"
 TOOLHIPPIE_CALENS = "toolhippie/calens:latest"
 
+secrets = {
+    "SQUISH_LICENSEKEY": "squish_license_server",
```

Comment:
I'm not sure we should expose this secret in PRs either... I think we could use the `push` trigger, too, to have it run on our own PRs.

Reply:
Yeah, that would be great. I have no idea how that can be done.
CC @xoxys

Reply:
Well... not that simple because then you will have two jobs running for every or the one for the PR `/ref/pull/**` itself and `/ref/head/**`... If you disable `/ref/pull/**` pull requests e.g. from external contributors via fork will not work but in that case you can just disable forks in drone.

Reply:
> disable forks in drone

That sounds totally acceptable to me. We should enable that for most secrets.

Reply:
Feel free to do so. As a repo admin, you can change this in the Drone UI for your repos on your own.

Reply:
I can do `disable forks` in drone if we want it

Reply:
Better we disable forks. It is possible to cleverly print the secret value
https://github.com/owncloud/enterprise/issues/5832#issuecomment-1628221384

Reply:
@xoxys I cannot log in to https://drone.owncloud.com. Who do I need to ask for access?